modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/html_util.ts

```diff
@@ -24,7 +24,7 @@ const MAP_HTML_CHARS_ESCAPED = {
 
 export {MAP_HTML_CHARS_ESCAPED};
 
-const MAP_HTML_CHARS_UNESCAPED = {};
+const MAP_HTML_CHARS_UNESCAPED: {[key: string]: string} = {};
```

Comment:
Similar to https://github.com/liferay-frontend/liferay-portal/pull/1379#discussion_r694536455 and on top of that, we also do:

```javascript
Object.entries(MAP_HTML_CHARS_ESCAPED).forEach(([char, escapedChar]) => {
	MAP_HTML_CHARS_UNESCAPED[escapedChar] = char;
});
```

Which also requires string-based key access. I get the feeling this could be reworked somehow if we wanted, but should be okay for now...